src/plotman/csv_exporter.py

```diff
@@ -0,0 +1,115 @@
+import csv
+import sys
+from dateutil.parser import parse as parse_date
+from plotman.log_parser import PlotLogParser
+from plotman.plotinfo import PlotInfo
+
+def export(logfilenames, save_to = None):
+    if save_to is None:
+        send_to_stdout(logfilenames)
+    else:
+        save_to_file(logfilenames, save_to)
+
+def save_to_file(logfilenames, filename: str):
+    with open(filename, 'w') as file:
+        generate(logfilenames, file)
+
+def send_to_stdout(logfilenames):
+    generate(logfilenames, sys.stdout)
+
+def header(writer):
```

Comment:
This is a pretty long list to just carefully keep aligned between the two places that the columns are and values are identified.  https://github.com/ericaltendorf/plotman/pull/299/files#diff-62b25901762e697ab64f8088b67b720707e694198e9d26ce3fafd3d08b14006fR262-R325 or such maybe but let's take a look again when the other files are here.

Reply:
Not sure if I understood. Mind to clarify?

Reply:
There are 34 items here and also 34 items specified around line 80.  It would be easy to get these out of order, or add a new item one place and not the other.  There would be no exception, I don't think, just misaligned column headings and data.  The linked code certainly isn't the only solution.  It was just one that I had coded recently (at that point).  Looking here again now I see that many of the fields are calculated from others.  This makes the linked code less obviously good.

Another thought, have a function that can create a `dict` from a `PlotInfo` object, maybe a method on `PlotInfo`, and use the keys from that as the header.  There is also a [`.writeheader()` method](https://docs.python.org/3/library/csv.html#csv.DictWriter.writeheader).  Then when you pass a dict to be written it will make sure the data goes in the matching column.